registry/registry.go

```diff
@@ -38,24 +40,24 @@ type Registry struct {
  * This passes http.DefaultTransport to WrapTransport when creating the
  * http.Client.
  */
```

Comment:
document that the context passed in is just used while setting up the client and is not stored